### PR TITLE
dev-vcs/git-cola: verbump to 3.9

### PR DESCRIPTION
**Pull request by stkw0**
Closes: https://bugs.gentoo.org/760582
Package-Manager: Portage-3.0.15, Repoman-3.0.2
Signed-off-by: David Roman <davidroman96@gmail.com>

### CODE REVIEW

**Review by bearsh — APPROVED** [review]
LGTM

File: dev-vcs/git-cola/git-cola-3.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+python_install_all() {
+	cd share/doc/${PN}/ || die
+	emake \
+		DESTDIR="${D}" \
+		docdir="${EPREFIX}/usr/share/doc/${PF}" \
+		prefix="${EPREFIX}/usr" \
+		install
+
+	# remove empty bin folder
+	rm -R "${ED}"/usr/share/git-cola/bin
```
Comment (juippis) [inline dev-vcs/git-cola/git-cola-3.9.ebuild:100]: Please add `|| die` here.

File: dev-vcs/git-cola/git-cola-3.9.ebuild [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+python_test() {
+	GIT_CONFIG_NOSYSTEM=true \
+	PYTHONPATH="${S}:${S}/build/lib:${PYTHONPATH}" LC_ALL="C.utf8" \
+	virtx nosetests --verbose --with-id --with-doctest \
+		--exclude=sphinxtogithub
+}
+
+src_install() {
+	distutils-r1_src_install
+}
```
Comment (juippis) [inline dev-vcs/git-cola/git-cola-3.9.ebuild:89]: This should be the default function, so you should be able to remove this block altogether.

**Review by juippis — COMMENTED** [review]
` dev-vcs/git-cola/git-cola-3.5-r1.ebuild → dev-vcs/git-cola/git-cola-3.5-r2.ebuild` what is this? Please remove that. Rest looks good.

**Comment by stkw0** [comment]
> ` dev-vcs/git-cola/git-cola-3.5-r1.ebuild → dev-vcs/git-cola/git-cola-3.5-r2.ebuild` what is this? Please remove that. Rest looks good.

Seems that the while rebasing I made a mistake and honestly I don't know why the r2 file appeared when it was not on ::gentoo.

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**